src/core/agent/services/multiSigService.ts

```diff
@@ -497,7 +497,7 @@ class MultiSigService extends AgentService {
       );
     } else {
       // Trigger the end role authorization if the operation is done
-      await Agent.agent.multiSigs.endRoleAuthorization(signifyName);
+      await this.endRoleAuthorization(signifyName);
```

Comment:
also test for this

Reply:
I have this in the previous commit already

Reply:
my mistake, its part of the same test title. all good thanks